src/js/main.js

```diff
@@ -444,39 +428,57 @@ export class Session {
         highScore[currentMode].playerName = scoreObj.playerName;
       }
     }
+
     this.store.updateAllHighScores(highScore[currentMode], currentMode);
+
+    const newScore = allScores[currentMode][allScores[currentMode].length - 1];
+
+    if (newScore.score == highScore[currentMode].bestScore) {
```

Comment:
Hmm, do you think using `>=` is more correct in this case?

What if high score is not equal but greater than the old best score?

```suggestion
    if (newScore.score >= highScore[currentMode].bestScore) {
```

What do you think?

Reply:
I added it below the condition where it loops and checks for best score, then it saves the `highScore[currentMode].bestScore ` in local storage. 

After that, I compare if the `newScore` is equal to whatever the best score saved in local storage is. :D